packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -22,7 +22,7 @@ import { promptAsync } from '../../prompts';
 import { getLastCommitMessageAsync } from '../../utils/git';
 
 export default class BranchPublish extends Command {
-  static hidden = true;
+  static hidden = false;
```

Comment:
keep as true

Reply:
oh my bad, didn't intend to push this. Thanks for catching it!